src/Microsoft.Performance.Toolkit.Engine/DataSourceSetExtensions.cs

```diff
@@ -11,9 +11,9 @@ namespace Microsoft.Performance.Toolkit.Engine
 {
     /// <summary>
     ///     Contains static (Shared in Visual Basic) methods for interacting
```

Comment:
can we get rid of "(Shared in Visual Basic)"

Reply:
I wrote these comments following those placed on extension method classes in the framework.